Hashtable 1.3.1 failed to build

C deps in `hashtables-1.3.1` library has a bug. Function `check_impl_specific` has parameters in the header, but in the implementation — no. 

```cabal
Building library for hashtables-1.3.1...
[ 1 of 12] Compiling Data.HashTable.Class ( src/Data/HashTable/Class.hs, dist/build/Data/HashTable/Class.o, dist/build/Data/HashTable/Class.dyn_o )
[ 2 of 12] Compiling Data.HashTable.Internal.Array ( src/Data/HashTable/Internal/Array.hs, dist/build/Data/HashTable/Internal/Array.o, dist/build/Data/HashTable/Internal/Array.dyn_o )
[ 3 of 12] Compiling Data.HashTable.Internal.IntArray ( src/Data/HashTable/Internal/IntArray.hs, dist/build/Data/HashTable/Internal/IntArray.o, dist/build/Data/HashTable/Internal/IntArray.dyn_o )

src/Data/HashTable/Internal/IntArray.hs:110:17: warning: [GHC-68441] [-Wdeprecations]
    In the use of ‘sizeofMutableByteArray’
    (imported from Data.Primitive.ByteArray):
    Deprecated: "use getSizeofMutableByteArray instead"
    |
110 | length (IA a) = A.sizeofMutableByteArray a `div` wordSizeInBytes
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^
[ 4 of 12] Compiling Data.HashTable.Internal.UnsafeTricks ( src/Data/HashTable/Internal/UnsafeTricks.hs, dist/build/Data/HashTable/Internal/UnsafeTricks.o, dist/build/Data/HashTable/Internal/UnsafeTricks.dyn_o )
[ 5 of 12] Compiling Data.HashTable.Internal.Linear.Bucket ( src/Data/HashTable/Internal/Linear/Bucket.hs, dist/build/Data/HashTable/Internal/Linear/Bucket.o, dist/build/Data/HashTable/Internal/Linear/Bucket.dyn_o )
[ 6 of 12] Compiling Data.HashTable.Internal.Utils ( src/Data/HashTable/Internal/Utils.hs, dist/build/Data/HashTable/Internal/Utils.o, dist/build/Data/HashTable/Internal/Utils.dyn_o )
[ 7 of 12] Compiling Data.HashTable.Internal.CheapPseudoRandomBitStream ( src/Data/HashTable/Internal/CheapPseudoRandomBitStream.hs, dist/build/Data/HashTable/Internal/CheapPseudoRandomBitStream.o, dist/build/Data/HashTable/Internal/CheapPseudoRandomBitStream.dyn_o )
[ 8 of 12] Compiling Data.HashTable.Internal.CacheLine ( src/Data/HashTable/Internal/CacheLine.hs, dist/build/Data/HashTable/Internal/CacheLine.o, dist/build/Data/HashTable/Internal/CacheLine.dyn_o )
[ 9 of 12] Compiling Data.HashTable.ST.Basic ( src/Data/HashTable/ST/Basic.hs, dist/build/Data/HashTable/ST/Basic.o, dist/build/Data/HashTable/ST/Basic.dyn_o )
[10 of 12] Compiling Data.HashTable.ST.Cuckoo ( src/Data/HashTable/ST/Cuckoo.hs, dist/build/Data/HashTable/ST/Cuckoo.o, dist/build/Data/HashTable/ST/Cuckoo.dyn_o )
[11 of 12] Compiling Data.HashTable.ST.Linear ( src/Data/HashTable/ST/Linear.hs, dist/build/Data/HashTable/ST/Linear.o, dist/build/Data/HashTable/ST/Linear.dyn_o )
[12 of 12] Compiling Data.HashTable.IO ( src/Data/HashTable/IO.hs, dist/build/Data/HashTable/IO.o, dist/build/Data/HashTable/IO.dyn_o )

cbits/default.c:201:6: error:
     error: conflicting types for ‘check_impl_specific’; have ‘void(int *, int *)’
      201 | void check_impl_specific(int* num_tests, int* num_errors) {
          |      ^~~~~~~~~~~~~~~~~~~
    |
201 | void check_impl_specific(int* num_tests, int* num_errors) {
    |      ^

In file included from cbits/default.c:3:0: error:

cbits/defs.h:33:6: error:
     note: previous declaration of ‘check_impl_specific’ with type ‘void(void)’
       33 | void check_impl_specific();
          |      ^~~~~~~~~~~~~~~~~~~
   |
33 | void check_impl_specific();
   |      ^
`gcc' failed in phase `C Compiler'. (Exit code: 1)
Error: [Cabal-7125]
Failed to build hashtables-1.3.1 (which is required by exe:semmc-ppc-synthdemo from semmc-ppc-0.1.0.0, exe:semmc-ppc-test from semmc-ppc-0.1.0.0 and others). See the build log above for details.
```

--- a/semmc/semmc.cabal
+++ b/semmc/semmc.cabal
@@ -2,7 +2,7 @@
 --  see http://haskell.org/cabal/users-guide/
 
 name:                semmc
-version:             0.1.0.1
+version:             0.1.0.0
 synopsis:            Semantics synthesis for machine code
 -- description:
 license:             BSD3
@@ -83,7 +83,7 @@ library
                        parsec >= 3.1 && < 3.2,
                        megaparsec >= 7 && < 10,
                        mtl >= 2.2 && < 2.4,
-                       hashtables (>= 1.2 && < 1.3.1) || (> 1.3.1 && < 1.5),
+                       hashtables >= 1.2 && < 1.5,
                        transformers >= 0.5.2 && < 0.7,
                        directory,
                        unliftio-core >= 0.2 && < 0.3,

--- a/semmc/semmc.cabal
+++ b/semmc/semmc.cabal
@@ -83,7 +83,7 @@ library
                        parsec >= 3.1 && < 3.2,
                        megaparsec >= 7 && < 10,
                        mtl >= 2.2 && < 2.4,
-                       hashtables >= 1.2 && < 1.4,
+                       hashtables >= 1.2 && < 1.4.3,
                        transformers >= 0.5.2 && < 0.7,
                        directory,
                        unliftio-core >= 0.2 && < 0.3,

--- a/semmc/semmc.cabal
+++ b/semmc/semmc.cabal
@@ -2,7 +2,7 @@
 --  see http://haskell.org/cabal/users-guide/
 
 name:                semmc
-version:             0.1.0.0
+version:             0.1.0.1
 synopsis:            Semantics synthesis for machine code
 -- description:
 license:             BSD3
@@ -83,7 +83,7 @@ library
                        parsec >= 3.1 && < 3.2,
                        megaparsec >= 7 && < 10,
                        mtl >= 2.2 && < 2.4,
-                       hashtables >= 1.2 && < 1.4.3,
+                       hashtables (>= 1.2 && < 1.3.1) || (> 1.3.1 && < 1.5),
                        transformers >= 0.5.2 && < 0.7,
                        directory,
                        unliftio-core >= 0.2 && < 0.3,

